pkg/manager/member/pd_scaler.go

```diff
@@ -142,6 +142,26 @@ func (psd *pdScaler) ScaleIn(tc *v1alpha1.TidbCluster, oldSet *apps.StatefulSet,
 	}
 	klog.Infof("pd scale in: delete member %s successfully", memberName)
 
+	// double check whether member deleted after delete member
```

Comment:
is [this](https://github.com/pingcap/tidb-operator/issues/2767) a bug in PD? if it is, add a link to the bug issue and comment it's a workaround solution. if it is not a bug, add a comment describing why the member may not be deleted after the deleting API request succeeds (deleted asynchronously?)

Reply:
I haven't found the bug yet and have no idea why the member haven't been deleted after deleting API success as it should be.

Reply:
"it should be after deleting API success"

if this is true, then it's a bug in PD, add the bug issue here. It will explain why this is required, otherwise, it's confusing

Reply:
sure https://github.com/pingcap/pd/issues/2541 updated.

Reply:
great!